src/index.html

```diff
@@ -11,8 +11,67 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100..900;1,100..900&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__container">
+        <div class="card__image">
+          <img
+            src="images/imac.jpeg"
+            alt="Product-name"
+            class="card__picture"
+          />
+        </div>
+
+        <div class="card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">Product code: 195434</div>
```

Comment:
You need to use the semantic tag for text, fix it everywhere
```suggestion
        <h3 class="card__description">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h3>

        <p class="card__code">Product code: 195434</p>
```